### DISCUSSION TITLE
Non-blocking error handler

### DISCUSSION THREAD

**Discussion by rjaros** [jooby-project/jooby#3402] [Q&A] [answered]
Is there a way to use non-blocking error handler - preferably with a Kotlin suspending function?

**Comment by edgar-espina-wpp** [comment]
the error handler will be non-blocking as long as you don't write blocking code. So yea, you want a kotlin suspend function. Please file a ticket for it.

**Comment by rjaros** [comment, marked as answer]
https://github.com/jooby-project/jooby/issues/3405
